src/pthread-main.js

```diff
@@ -126,7 +126,7 @@ this.onmessage = function(e) {
       STACK_MAX = STACK_BASE + e.data.stackSize;
       assert(STACK_BASE != 0);
       assert(STACK_MAX > STACK_BASE);
-      establishStackSpace(e.data.stackBase, e.data.stackBase + e.data.stackSize);
+      Module.establishStackSpace(e.data.stackBase, e.data.stackBase + e.data.stackSize);
```

Comment:
These could be `Module['..']` which is closure compiler friendly. I don't think we run closure on this file now, and maybe that isn't too important for it (as a small file), but it would also be nice for consistency.